<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Lp-ANKUSH26" failures="2" tests="2" name="Test(failed)" time="0.094" errors="0" timestamp="2025-05-16T10:09:40 IST">
  <testcase classname="testNG_Failed_TestCases.Class_A" name="method1" time="0.026">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG_Failed_TestCases.Class_A.method1(Class_A.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- method1 -->
  <testcase classname="testNG_Failed_TestCases.Class_B" name="method3" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG_Failed_TestCases.Class_B.method3(Class_B.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- method3 -->
  <testcase classname="testNG_Failed_TestCases.Class_A" name="method2" time="0.0">
    <ignored/>
  </testcase> <!-- method2 -->
  <testcase classname="testNG_Failed_TestCases.Class_B" name="method4" time="0.0">
    <ignored/>
  </testcase> <!-- method4 -->
</testsuite> <!-- Test(failed) -->
